x-pack/test/api_integration/deployment_agnostic/apis/observability/incident_management/suggested_dashboards.ts

```diff
@@ -0,0 +1,1227 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import rawExpect from 'expect';
+import expect from '@kbn/expect';
+import { join } from 'path';
+import { cleanup, generate, Dataset, PartialConfig } from '@kbn/data-forge';
+import type { RoleCredentials } from '@kbn/ftr-common-functional-services';
+import { Aggregators } from '@kbn/observability-plugin/common/custom_threshold_rule/types';
+import { COMPARATORS } from '@kbn/alerting-comparators';
+import { OBSERVABILITY_THRESHOLD_RULE_TYPE_ID } from '@kbn/rule-data-utils';
+import { DeploymentAgnosticFtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const CUSTOM_THRESHOLD_RULE_ALERT_INDEX = '.alerts-observability.threshold.alerts-default';
+  const esClient = getService('es');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const supertestWithAuth = getService('supertest');
+  const kibanaServer = getService('kibanaServer');
+  const esDeleteAllIndices = getService('esDeleteAllIndices');
+  const logger = getService('log');
+  const alertingApi = getService('alertingApi');
+  const samlAuth = getService('samlAuth');
+  const spacesService = getService('spaces');
+  const config = getService('config');
+  const isServerless = config.get('serverless');
+  const expectedConsumer = isServerless ? 'observability' : 'logs';
+  const scheduleInterval = isServerless ? '1m' : '10s';
+
+  let dataForgeConfig: PartialConfig;
+  let editorUser: RoleCredentials;
+  let dataForgeIndices: string[];
+  let ruleId: string;
+  const SPACE_ID = 'test_space';
+
+  describe('Suggested dashboards', function () {
+    this.tags(['skipCloud', 'skipMKI']);
+    before(async () => {
+      editorUser = await samlAuth.createM2mApiKeyWithRoleScope('editor');
+      await spacesService.delete(SPACE_ID);
+      await kibanaServer.savedObjects.cleanStandardList();
+      await spacesService.create({
+        id: SPACE_ID,
+        name: 'Test Space',
+        disabledFeatures: [],
+        color: '#AABBCC',
+      });
+      dataForgeConfig = {
+        schedule: [
+          {
+            template: 'bad',
+            start: 'now-15m',
+            end: 'now+5m',
+          },
+        ],
+        indexing: {
+          dataset: 'fake_stack' as Dataset,
+          eventsPerCycle: 1,
+          interval: 10000,
+          alignEventsToInterval: true,
+        },
+      };
+      dataForgeIndices = await generate({ client: esClient, config: dataForgeConfig, logger });
+      await alertingApi.waitForDocumentInIndex({
+        indexName: dataForgeIndices.join(','),
+        docCountTarget: 500,
+      });
+
+      const { body } = await supertestWithAuth
+        .post('/api/saved_objects/_import')
+        .set(samlAuth.getInternalRequestHeader())
+        .attach('file', join(__dirname, './fixtures/dashboards_default_space.ndjson'))
```

Comment:
@dominiqueclarke Do you mind sharing how you got this file generated?

Reply:
I added some lens panels to the dashboards created by data-forge using the fake stack template and --install-kibana-assets. Then I exported them via kibana saved object export functionality.